app/teachingCall/teachingCallStatus/controllers/ModalAddInstructorsCtrl.js

```diff
@@ -30,18 +31,49 @@ teachingCallApp.controller('ModalAddInstructorsCtrl', ['$scope', '$rootScope', '
 		'03': 'Spring Quarter'
 	};
 
-	$scope.senateGroup = angular.copy(state.eligible.senate);
-	$scope.federationGroup = angular.copy(state.eligible.federation);
-	$scope.lecturerGroup = angular.copy(state.eligible.lecturer);
-
-	$scope.startTeachingCallConfig.invitedInstructors = $scope.senateGroup.concat($scope.federationGroup);
-	$scope.startTeachingCallConfig.invitedInstructors = $scope.startTeachingCallConfig.invitedInstructors.concat($scope.lecturerGroup);
+	$scope.startTeachingCallConfig.invitedInstructors = state.calculations.instructorsEligibleForCall;
 	$scope.startTeachingCallConfig.invitedInstructors = _array_sortByProperty($scope.startTeachingCallConfig.invitedInstructors, "lastName");
 
-	$scope.startTeachingCallConfig.invitedInstructors.forEach(function(slotInstructor) {
-		slotInstructor.invited = false;
+	$scope.startTeachingCallConfig.invitedInstructors.forEach(function(instructor) {
+		instructor.invited = false;
 	});
 
+	$scope.areThereInstructorsOfType = function(instructorTypeId) {
```

Comment:
Please rename this method. Perhaps instructorTypeUsed() ?

Reply:
I like instructorTypeUsed() is better, I puzzled over areThereInstructorsOfType() but couldn't think of anything.